Modules/Core/Common/include/itkImageScanlineConstIterator.h

```diff
@@ -265,7 +265,7 @@ class ITK_TEMPLATE_EXPORT ImageScanlineConstIterator : public ImageConstIterator
 // Deduction guide for class template argument deduction (CTAD).
 template <typename TImage>
 ImageScanlineConstIterator(SmartPointer<TImage>, const typename TImage::RegionType &)
-  ->ImageScanlineConstIterator<std::remove_const_t<TImage>>;
+  -> ImageScanlineConstIterator<std::remove_const_t<TImage>>;
```

Comment:
Glad to see that the new clang format "understands" the syntax of those CTAD deduction guides 👍